src/main/java/uk/gov/hmcts/reform/prl/services/caseflags/FlagsService.java

```diff
@@ -107,6 +107,16 @@ private static List<String> getAllFlagsToValidate() {
             "caOtherParty3InternalFlags",
             "caOtherParty4InternalFlags",
             "caOtherParty5InternalFlags",
+            "caApplicantBarrister1InternalFlags",
```

Comment:
Pls add CA applicant and respondent external flags and DA applicant and respondent external and internal flags